atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/service/AsyncTransactionService.java

```diff
@@ -47,5 +48,40 @@ public interface AsyncTransactionService {
      * @return {@link ListenableFuture} containing the map from a transaction start timestamp to transaction commit
      * timestamp, possibly missing entries if relevant transactions have not committed yet
      */
+    @Deprecated
```

Comment:
as in the synchronous `TransactionService`, it should be fairly obvious, but it's probably worth telling people to use the respective `V2` methods.